docs/massivekb.md

```diff
@@ -4,7 +4,30 @@ The MassIVE Knowledge Base (MassIVE-KB) is a set of peptide spectral libraries.
 
 ## Browsing Libraries
 
-You can browser MassIVE-KB libraries to explore what is in them, and what is the provenance. 
+You can browser MassIVE-KB libraries to explore what is in them, and what is the provenance. Here is a view of the status page of a massive-kb library:
```

Comment:
1. "browse", not "browser"
2. "massive-kb" should be capitalized everywhere as "MassIVE-KB".